### ISSUE TITLE
Suplimentary static pages

### ISSUE THREAD

**Issue by auxfuse** [auxfuse/hackathon-git-labs#52]
Grouping these into:
* Privacy Policy
* Accessibility
* SEO/Cookie Statement
* Other? - whatever is necessary (consider linking to Code Institute Website?)

**Comment by auxfuse** [comment]
@manni8436 😄

**Comment by auxfuse** [comment]
@andrewdempsey2018 @manni8436 going to split this in two ✂️ 

@manni8436 can you do the Privacy Policy & Accessibility pages? And we can then get in the SEO & Cookies policy later along with other pieces from @andrewdempsey2018 🤜

**Comment by manni8436** [comment]
Yep no worries, I can get that done @auxfuse 😊, I'll start on it tomorrow afternoon after work 🤓
